en_US/network/crl.md

```diff
@@ -0,0 +1,34 @@
+### CRL Check
+
+{% emqxce %}
+
+Since EMQX 5.0.22, Certification Revocation List (CRL) Check is supported for MQTT SSL listeners.  Note that those do not include Secure WebSocket nor QUIC listeners: only listeners of type `ssl` support this feature.
```

Comment:
```suggestion
Since EMQX 5.0.22, CRL(Certification Revocation List) Check is supported for MQTT SSL listeners.  Note that those do not include Secure WebSocket nor QUIC listeners: only listeners of type `ssl` support this feature.
```